.travis.yml

```diff
@@ -1,56 +0,0 @@
-sudo: required
-
-addons:
-  postgresql: "9.6"
-  apt:
-    packages:
-      - postgresql-contrib-9.6
-      - libxmlsec1-dev
-      - libxml2-dev
-
-services:
-  - postgresql
-
-language: python
-
-python:
-  - "2.7"
-
-cache: pip
-
-git:
-  submodules: false
-
-before_install:
-  # Access the submodule via https instead of default SSH
-  - git config submodule.core.url https://github.com/NYPL-Simplified/server_core.git
-  - git config submodule.docker.url https://github.com/NYPL-Simplified/circulation-docker.git
-  - git submodule update --init --recursive
-
-install:
-  - pip install --upgrade pip
-  - pip install -r requirements.txt
-  - python -m textblob.download_corpora
-
-before_script:
-  - psql -c 'create user simplified_test;' -U postgres
-  - psql -c 'create database simplified_circulation_test;' -U postgres
-  - psql -c 'grant all privileges on database simplified_circulation_test to simplified_test;' -U postgres
-  - cd docs && make html && cd ..
-
-env:
-  global:
-    - SIMPLIFIED_TEST_DATABASE="postgres://simplified_test:test@localhost:5432/simplified_circulation_test"
-
-script:
-  - ./verbose-test
-
-deploy:
-  provider: pages
-  skip_cleanup: true
-  keep-history: true
-  github_token: $GITHUB_TOKEN
-  on:
-    branch: main
-  local_dir: docs/build/html/
```

Comment:
I didn't move this deployment to github actions, since I'm not sure all the details here. Someone from NYPL will probably have to write the action for this.

Reply:
That will be @EdwinGuzman, though I can figure it out if necessary. I'm okay with doing this in between merging this branch and doing the Python 3 conversion.

Reply:
It looks pretty simple: https://github.com/marketplace/actions/deploy-to-github-pages